pandas/tests/frame/conftest.py

```diff
@@ -0,0 +1,243 @@
+import pytest
+
+import numpy as np
+
+from pandas import compat
+import pandas.util.testing as tm
+from pandas import DataFrame, date_range, NaT
+
+
+# This module is the start of transitioning from attributes of
```

Comment:
actually no, let's change them to something reasonable here the form, its easier to change them once and then adjust on file migration.

``int_frame``, ``ts_frame`` etc is reasonable, e.g. _ separate and they all have the word frame in them.